app/routes/people/person/positions/mandatory.js

```diff
@@ -1,14 +1,14 @@
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 
-export default class PeoplePersonPositionsPositionRoute extends Route {
+export default class PeoplePersonPositionsMandatoryRoute extends Route {
   @service store;
 
   async model() {
     let person = this.modelFor('people.person');
 
     let { positionId: mandatoryId } = this.paramsFor(
```

Comment:
I'm not sure I understood it, how it should be? because than I need the positionId.

Reply:
Yea, so the mandatory route has a `positionId` dynamic segment. This meant that the model hook of that route will receive the value for that as the first argument to the model method.

Similar to this for the person route:
https://github.com/lblod/frontend-contact-hub/blob/aaa5c7f070120a5b97799ec50c8702530e5eed30/app/routes/people/person.js#L4

So instead of using `this.paramsFor` you can get the id from that first argument instead:

`async model({ positionId: mandatoryId }) {...`

paramsFor is only needed if you want to retrieve params for a parent route.